src/server/tests.rs

```diff
@@ -1451,7 +1451,9 @@ async fn test_completion_resolve() {
 async fn test_package_completion() {
     let fluxscript = r#"import "sql"
 
-sql."#;
+sql.
+// ^
```

Comment:
This is interesting. Doesn't this change the test?

Reply:
Probably the position was of-by-one by mistake and it happened to work either way.

Reply:
Right, but couldn't we just move the cursor position? We've added lines to the test data now, right?

Reply:
Yes, but that doesn't change what we are testing for. If anything, this shows that specifying line/column manually is error prone. Specifying it inline makes it clearer both for the reader and writer where the cursor actually is.

Reply:
Oh, I had to go look at the source for `position_of`. Now I understand, but I'll admit I'm not a huge fan of that. Yes, I think specifying positions manually is error prone, but this is a test, and I'd much rather specify it manually for the purpose of the test.

Reply:
Why do you want to specify it manually? It is both harder for the writer to specify the correct position and for the reader to understand which position that is being tested (at lest for me, I hate having to count the lines and characters to identify the position that is tested, and then I have to double check whether it is 0 or 1 indexed).

Reply:
If it is the same concern as https://github.com/influxdata/flux-lsp/pull/384#discussion_r791936964 we need to also remove `create_server` and `open_file` to be consistent (and probably `expect_test` as well).